<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GdalToolsWidget</class>
 <widget class="QWidget" name="GdalToolsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>153</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Sieve</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Input file:</string>
       </property>
       <property name="buddy">
        <cstring>selectInputFileButton</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QComboBox" name="inputLayerCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::NoInsert</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectInputFileButton">
         <property name="text">
          <string>Select...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Output file:</string>
       </property>
       <property name="buddy">
        <cstring>selectOutputFileButton</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="outputFileEdit">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="selectOutputFileButton">
         <property name="text">
          <string>Select...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="thresholdSpin">
       <property name="maximum">
        <number>65000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="connectionsCombo">
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="thresholdCheck">
       <property name="text">
        <string>&amp;Threshold:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="connectionsCheck">
       <property name="text">
        <string>&amp;Pixel connections:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
